src/hooks/useQuickstartCloseOnUnmount.ts

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { useChrome } from '@redhat-cloud-services/frontend-components/useChrome';
+
+const useQuickstartCloseOnUnmount = () => {
```

Comment:
Did we get this naming convention from somewhere? Seems odd to have `unmount` in a hook name

Reply:
Yeah.  I copied that form hac-dev.  It looks like they have renamed as I'm not seeing it now https://cs.github.com/?scopeName=All+repos&scope=&q=useQuickstartCloseOnUnmount